redwood-treehouse-composeui/src/main/kotlin/app/cash/redwood/treehouse/composeui/TreehouseComposeView.kt

```diff
@@ -23,26 +23,30 @@ import androidx.compose.foundation.layout.Column
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.platform.ComposeView
-import app.cash.redwood.LayoutModifier
-import app.cash.redwood.protocol.ChildrenDiff.Companion.RootChildrenTag
-import app.cash.redwood.protocol.EventSink
-import app.cash.redwood.protocol.PropertyDiff
-import app.cash.redwood.protocol.widget.DiffConsumingWidget
 import app.cash.redwood.treehouse.TreehouseApp
 import app.cash.redwood.treehouse.TreehouseView
 import app.cash.redwood.widget.Widget
 import app.cash.redwood.widget.compose.ComposeWidgetChildren
-import kotlinx.serialization.json.JsonArray
 
 @SuppressLint("ViewConstructor")
 public class TreehouseComposeView<T : Any>(
   context: Context,
   private val treehouseApp: TreehouseApp<T>,
   public val widgetFactory: Widget.Factory<@Composable () -> Unit>,
 ) : FrameLayout(context), TreehouseView<T> {
-  public val composeView: ComposeView = ComposeView(context)
+  private val _children = ComposeWidgetChildren()
+  override val children: Widget.Children<*> get() = _children
 
   init {
+    val composeView = ComposeView(context)
```

Comment:
This class should extend `AbstractComposeView` rather than using a pointless `FrameLayout`. Future change.